ci/push_docker_multiplatform_manifest.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if ! docker buildx ls | grep -q arm ; then
+    echo "Your Buildx seems to lack ARM architecture support"
+    echo
+    docker buildx ls
+    exit 1
+fi
```

Comment:
does it mean I cannot build other architecture unless I have arm support?

Reply:
I could improve this checking the params to this scripts against output of `docker buildx ls` 👍